monitoring/configuration/grafana.ini

```diff
@@ -24,9 +24,9 @@ enabled = true
 ; org_role = Admin
 org_role = Viewer
 
-;[security]
-;admin_user = ocr
-;admin_password = ocr
+[security]
+admin_user = wakusim-user
```

Comment:
No need to modify the config file, env variable can override the config https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables.
Also i'm not sure the config file with env var inside will work

Reply:
I did test it locally and it worked. Good to know. I have updated the file to just be the default values. (https://github.com/waku-org/waku-simulator/pull/56/commits/9f93120a631f54ecb944637da32861f8ab32da27)